scripts/scripts.js

```diff
@@ -0,0 +1,93 @@
+const app = {};
+
+// asynchronous fetch of local JSON data
+app.getData = async () => {
+    await fetch("./resources/resume.json")
+    .then(response => response.json())
+    .then(data => {
+        // call helper function to display all user data
+        app.displayAllUserData(data);
+    })
+    .catch(error => console.error(error));
+}
+
+// insert basic user information to HTML element for contact information
+app.displayContactData = (userData, targetProperty) => {
+    const { firstName, lastName, email, github, location } = userData[targetProperty];
+
+    const htmlToAppend = `
+    <h1>${firstName} ${lastName}</h1>
+    <ul>
+    <li>${location.city}, ${location.country}</li>
+    <li>${email}</li>
+    <li><a href=${github}>Github</a></li>
+    </ul>
+    `; 
+    
+    // insert basic user info into HTML element
+    document.querySelector(`#${targetProperty}`).innerHTML = htmlToAppend;
```

Comment:
This works. A heads up: this could cause JS injection errors if you didn't control the source data. A bit of a security issue in that sense.